lib/service.rb

```diff
@@ -72,7 +72,7 @@ def plist?
     # Returns `true` if the service is loaded, else false.
     def loaded?
       # TODO: find replacement for deprecated "list"
-      `#{ServicesCli.launchctl} list | grep '#{label}$' 2>/dev/null`.chomp =~ /#{label}\z/
+      quiet_system ServicesCli.launchctl, "list", "'#{label}'"
```

Comment:
Does this work with older macOS versions?

Reply:
I can find it in older generated manpages (and it is a legacy command). But I am not sure.

Reply:
First data point is in: it does on Mojave

Reply:
@SMillerDev That's good enough for me!